cloudnet-modules/cloudnet-labymod/src/main/java/eu/cloudnetservice/cloudnet/ext/labymod/node/CloudNetLabyModModule.java

```diff
@@ -19,7 +19,7 @@
 
     private static final Map<String, Boolean> DEFAULT_LABY_MOD_PERMISSIONS = new HashMap<>();
     private static final LabyModPermissionConfig LABY_MOD_PERMISSION_CONFIG = new LabyModPermissionConfig(false, DEFAULT_LABY_MOD_PERMISSIONS);
-    private static final LabyModBannerConfig BANNER_CONFIG = new LabyModBannerConfig(false, "https://dl.hdskins.de/CloudNet.png");
+    private static final LabyModBannerConfig BANNER_CONFIG = new LabyModBannerConfig(false, "http://dl.cloudnetservice.eu/data/minecraft/CloudNet-LabyMod-Banner.png");
```

Comment:
Please use https here

Reply:
This will not work unfortunately as the Minecraft launcher ships a very old Java version, which does not support Lets Encrypt certificates (what we are using).

Reply:
Ah right, the client downloads the banner (isn't there a redirect to https?)